ague26/stags-game-of-games
# Adding Cypress and a basic end-to-end smoke test

#### What does this PR do
* Adds cypress as a dev dependency
* I wrote a test call `app.spec.js`

#### How to run the test

Once this is merge, do a `npm install`

There's a few commands:
* `npm run cypress:open` (this will open Cypress but also make sure that you have ran `npm start` before)
* `npm run cypress:run:ci` (this runs on travis, you don't really have to worry about this)

<img width="1505" alt="Screen Shot 2021-03-27 at 8 54 34 PM" src="https://user-images.githubusercontent.com/8345650/112739057-04bae180-8f3f-11eb-8167-a01db12ea331.png">
